### ISSUE TITLE
Infoclick (new version): Issue with layers containing over 100 attributes.

### ISSUE THREAD

**Issue by eri-blan** [hajkmap/Hajk#1146]
**Describe the bug**
The problem seems to occur when there is more than a 100 attributes in the GetFeatureInfo response. A pageSize is given according to the number of attributes and if it is over a hardcoded value of a 100, an error is thrown and the whole screen becomes white when clicking on a result, or if there is only one result, it happens immediately when clicking on the map.

Below is where the error occurs:

export const useDataGridProps = inProps => {
  if (inProps.pageSize > MAX_PAGE_SIZE) {
    throw new Error(`'props.pageSize' cannot exceed 100 in DataGrid.`);
  }

**To Reproduce**
Try the WMS service:
https://resource.sgu.se/service/wms/130/berggrund-50-250-tusen?service=WMS&REQUEST=GetCapabilities
Using the layer below:
SE.GOV.SGU.BERG.GEOLOGISK_ENHET.YTA.50K

(Note that the getmap-link is different from the service link)

**Comment by jacobwod** [comment]
Ahh… check [this](https://mui.com/x/react-data-grid/pagination/#size-of-the-page) out.

> Size of the page
The MIT DataGrid is limited to pages of up to 100 rows. If you want larger pages, you will need to upgrade to [Pro plan](https://mui.com/x/introduction/licensing/#pro-plan) or above.

**Comment by eri-blan** [comment]
> Ahh… check [this](https://mui.com/x/react-data-grid/pagination/#size-of-the-page) out.
> 
> > Size of the page
> > The MIT DataGrid is limited to pages of up to 100 rows. If you want larger pages, you will need to upgrade to [Pro plan](https://mui.com/x/introduction/licensing/#pro-plan) or above.

Yes, I might be wrong but what I understood is stated is that the page size is limited to a 100 rows, but that there will be pagination by default in the free version, whereas in the pro version a singular page can exceed 100 rows. I think maybe that if it is possible to paginate the attribute information, it is better than an error that renders the whole application unusable.

**Comment by jacobwod** [comment]
I couldn't use the layer you supply due to CORS restrictions. Do you have a local example on the server you emailed me about previously? 

Anyway, I did try with another layer and yes, the maximum items per page is 100, but I can see the total and paginate as well:
![Skärmavbild 2022-09-26 kl  12 56 13](https://user-images.githubusercontent.com/110222/192259878-cb8dc4bd-8a5d-4727-b760-35276d6f7cf3.png)

Isn't this the case for you?
